aspnetcore/grpc/dotnet-grpc.md

```diff
@@ -45,20 +56,20 @@ dotnet grpc add-file [options] <files>...
 
 | Argument | Description |
 |-|-|
-| files | The protobuf file reference(s). These can be a path to glob for local protobuf file(s). |
+| files | The protobuf file references. These can be a path to glob for local protobuf files. |
```

Comment:
I lifted these from the CLI help text directly. Is the recommendation to update the help text? @shirhatti what do you think?